std/http/file_server.ts

```diff
@@ -41,6 +40,25 @@ const CORSEnabled = serverArgs.cors ? true : false;
 const target = posix.resolve(serverArgs._[1] ?? "");
 const addr = `0.0.0.0:${serverArgs.port ?? serverArgs.p ?? 4500}`;
 
+const MEDIA_TYPES: Record<string, string> = {
+  ".md": "text/markdown",
+  ".html": "text/html",
+  ".htm": "text/html",
+  ".json": "application/json",
+  ".map": "application/json",
+  ".txt": "text/plain",
+  ".ts": "application/typescript",
```

Comment:
@ry I just tried it and found that using `application/typescript`, the browser will download the file directly, I think that `.ts` should be mapped to "application/javascript" or "application/typescript; charset = UTF-8"